### PR TITLE
[hammurabi] Update to match the latest baseline

### PR DESCRIPTION
**Pull request by gabor-boros**
## Description
Below you can find the executed laws and information about them.

### Implement community standards
Recommended community standards: [https://opensource.guide/](https://opensource.guide/)

<details>
<summary>Passed rules</summary>

* License must be Apache 2.0
* Code of conduct must be set
* Contributing guidelines must be set
* Remove .github directory if exists
* The .github folder must contain necessary files
</details>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (zicklag) [inline LICENSE:1]: We already have a `LICENSE-APACHE` file, so is this necessary?

**Review by zicklag — COMMENTED** [review]
This should be rebased onto master right?

**Comment by gabor-boros** [comment]
Hello @zicklag 👋 

This was an automated PR across 14+ repos. As you mentioned, you already have an Apache 2.0 license so that change is not needed at all. 

Which branch should be the default for this repo? `next` or `master`? Also, what's the purpose of the `next` branch? Last time I checked this repository it seemed to me that the development stopped on the `next` branch and continued on the `master` on. If that's the case, who is the current maintainer of this driver? 😇

**Comment by zicklag** [comment]
Hi @gabor-boros :wave: 

Ah, that makes sense. :) I was wondering what `[hammurabi]` was.

So the `next` branch is an incomplete attempt at updating the driver with the latest `async` Rust and other improvements, but it is really not ready and may need to be semi-re-done as the `async` ecosystem has developed a lot since then.

The `master` branch is the currently released version that does *work*, but definitely could use some love and has some performance issues. Right now our maintenance changes should all go to `master`.

I'm a new collaborator on the project that got on so that I could merge some PRs by my colleague that needed to use this but with some dependency updates. I'm wanting to modernize and update the driver to the latest `async` Rust, add more examples, and create actual Rust API documentation ( not just JavaScript documentation ), but until I can find a more substantial use-case for my work I won't be able to spend much time on it.

---

@rushmorem Do you think we could change the default branch of the repo to `master` instead of `next`. It is rather confusing, especially to newcomers who take a look at the examples dir, which don't work for the released version.

**Comment by gabor-boros** [comment]
Ok, I forgot this PR. I’ll do this this week.

**Pull request closed without merging**